packages/prepare/src/Prepare.ts

```diff
@@ -67,7 +67,7 @@ function uploadGraphics(renderer: AbstractRenderer | BasePrepare, item: IDisplay
     // if its not batchable - update vao for particular shader
     if (!geometry.batchable)
     {
-        (renderer as Renderer).geometry.bind(geometry, (item as any)._resolveDirectShader());
+        (renderer as Renderer).geometry.bind(geometry, (item as any)._resolveDirectShader((renderer as Renderer)));
```

Comment:
Why is `item` cast as any? We should have caught this issue way sooner.

Reply:
Some kind of legacy?

Reply:
I think was just an issue with with this package being converted to TypeScript #6481. We should followup with a PR, that create an Interface for batchable objects.